app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt

```diff
@@ -49,6 +52,16 @@ class FavoriteFragment : Fragment() {
             thisActivity.supportActionBar?.setDisplayHomeAsUpEnabled(false)
         }
 
+        rootView.findSomethingToDo.setOnClickListener {
+            val builder = NavOptions.Builder()
+            builder.setPopUpTo(R.id.eventsFragment, false)
+            builder.setEnterAnim(R.anim.slide_in_right)
+            builder.setExitAnim(R.anim.slide_out_left)
+            builder.setPopEnterAnim(R.anim.slide_in_left)
+            builder.setPopExitAnim(R.anim.slide_out_right)
+            Navigation.findNavController(rootView).navigate(R.id.searchFragment, null, builder.build())
```

Comment:
okay, just Remove duplication of imports. Use `findNavController` instead of `Navigation.findNavController` and remove the Navigation import.